cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -90,7 +87,7 @@ define([
         delete window.course;
     });
 
-    describe('GroupConfigurationDetails', function() {
+    describe('GroupConfigurationDetailsView', function() {
```

Comment:
I followed the existing pattern of matching the `describe` blocks' names to the name of the view they're testing. Would it suffice to include this information as a comment, or would you rather see the string itself change?